examples/README.md

```diff
@@ -61,8 +60,9 @@ aspNetAppTargetFramework="net6.0" ./run-example.sh
 
 ### BindingRedirect
 
-Example client app that uses .NET Framework binding redirects to handle assembly version conflict
```

Comment:
In this repo Sample application means application used in tests. Example application is app just to illustrate something, placed in different folder.

I hope you find a way to rephrase to differentiate the usage. Alternatively, we can try to change the code if you have better names :)

Reply:
#484